kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc

```diff
@@ -163,7 +163,7 @@ Before getting started, you should have:
 
 * VPC, subnet, routing, and security group configurations sufficient to allow HTTPS access to the deployment host interface, as well as open access between the ephemeral appliance and the subnets where your EKS cluster will be configured
 
-* Route53 registered domain
+* Route53 registered domain under the currently logged in AWS account. If leveraging a Route53 domain from a different account, please refer to https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html[Delegate access across AWS accounts]
```

Comment:
Consider these few minor changes:

Route53 registered domain under the currently AWS account.  If you need to leverage a Route53 domain from a different AWS account, please refer to https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html[Delegate access across AWS accounts]